lib/google_maps.ex

```diff
@@ -1861,10 +1876,10 @@ defmodule GoogleMaps do
       true
 
   """
-  @spec get(String.t, options()) :: Response.t()
+  @spec get(String.t(), options()) :: Response.t()
   def get(endpoint, params) do
     Request.get(endpoint, params)
-    |> Response.wrap
+    |> Response.wrap()
   end
 
   @spec coordinate(coordinate()) :: binary()
```

Comment:
Do we need to change this function spec too?

Reply:
No that one is fine. Let it do the conversion if the pattern matches, otherwise return the string

Reply:
Sorry. I meant the spec :)